doc/content/gettingstarted/cli/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: "Getting Started with the Command-line Interface"
```

Comment:
Maybe this should be only `Command-line Interface` ? We are stuttering `Getting Started - Getting started with the Command-line Interface`